src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/form.html.twig

```diff
@@ -281,10 +281,21 @@
                     }
 
                     // attach listener for expanding/collapsing the target
-                    buttons[i].addEventListener('click', function (e) {
-                        toggle(this);
+                    Sfjs.addEventListener(buttons[i], 'click', function (e) {
+                        if (!e) {
+                            e = window.event;
+                        }
+
+                        if (e.target) {
+                            target = e.target;
```

Comment:
you are missing the variable delcaration for this variable in the function. It should be a local variable